azure/timer.ts

```diff
@@ -12,39 +12,62 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import * as azure from "@pulumi/azure";
+import * as subscription from "@pulumi/azure-serverless/subscription";
 import { timer } from "@pulumi/cloud";
 import * as pulumi from "@pulumi/pulumi";
 import { RunError } from "@pulumi/pulumi/errors";
-// import { Function } from "./function";
+import * as shared from "./shared";
 
 export function interval(name: string, options: timer.IntervalRate, handler: timer.Action,
                          opts?: pulumi.ResourceOptions): void {
```

Comment:
@lukehoban We may want to consider having the cloud.timer functions return a Resource.  It feels weird to have fire-and-forget functions that make resources but don't give them back to the caller.

Reply:
Yes - this is a good idea.  Want to open an issue to track this at least?

It's a bit of a shame that it would have to be loosely typed as `Resource` at the `@pulumi/cloud` layer, but that would at least allow it to be passed in as a dependency if needed.